Packs/iDefense/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Accenture CTI",
     "description": "Accenture Security",
     "support": "partner",
-    "currentVersion": "3.2.2",
+    "currentVersion": "3.3.0",
```

its a revision change does not have to be minor
```suggestion
    "currentVersion": "3.2.3",
```

---

Done